recipes/busybox/busybox.inc

```diff
@@ -75,6 +75,11 @@ do_compile_binfmt_flat_fixup() {
 
 require busybox-install.inc
 
+inherit s6rc
+SRC_URI:>USE_s6rc = " file://rtc.up file://rtc.down"
+S6RC_ONESHOT_SERVICES:>USE_busybox_hwclock = " rtc"
+RECIPE_FLAGS += "rtc_s6rc_dependencies"
```

Comment:
This is used by s6rc.oeclass, allowing for (fx. DISTRO) configuration of service dependencies for rtc service.